examples/npe_model/waveform_dataset_settings.yaml

```diff
@@ -0,0 +1,40 @@
+domain:
+  type: FrequencyDomain
+  f_min: 20.0
+  f_max: 1024.0
+  delta_f: 0.125  # Expressions like 1.0/8.0 would require eval and are not supported
+
+waveform_generator:
+  approximant: IMRPhenomXPHM  # SEOBNRv4PHM
+  f_ref: 20.0
+  # f_start: 15.0  # Optional setting useful for EOB waveforms. Overrides f_min when generating waveforms.
+
+# Dataset only samples over intrinsic parameters. Extrinsic parameters are chosen at train time.
+intrinsic_prior:
+  mass_1: bilby.core.prior.Constraint(minimum=10.0, maximum=120.0)
+  mass_2: bilby.core.prior.Constraint(minimum=10.0, maximum=120.0)
+  chirp_mass: bilby.gw.prior.UniformInComponentsChirpMass(minimum=15.0, maximum=150.0)
+  mass_ratio: bilby.gw.prior.UniformInComponentsMassRatio(minimum=0.125, maximum=1.0)
+  phase: default
+  a_1: bilby.core.prior.Uniform(minimum=0.0, maximum=0.99)
+  a_2: bilby.core.prior.Uniform(minimum=0.0, maximum=0.99)
+  tilt_1: default
+  tilt_2: default
+  phi_12: default
+  phi_jl: default
+  theta_jn: default
```

Comment:
I can update this, though which aligned spin model? I think SEOBNRv4 makes sense since it's the simplest aligned spin model that I've tested. Though I haven't trained a pure NPE network.